evap/evaluation/tests/test_views.py

```diff
@@ -156,17 +156,34 @@ def test_requires_login(self):
 
     def test_save_settings(self):
         user = baker.make(UserProfile)
-        page = self.app.get(self.url, user=self.responsible, status=200)
+        page = self.app.get(self.url, user=self.responsible)
         form = page.forms["settings-form"]
         form["delegates"] = [user.pk]
-        form.submit()
+        form.submit(name="operation", value="delegates")
 
         self.responsible.refresh_from_db()
         self.assertEqual(list(self.responsible.delegates.all()), [user])
 
     def test_view_settings_as_non_editor(self):
         user = baker.make(UserProfile, email="testuser@example.com")
-        page = self.app.get(self.url, user=user, status=200)
+        page = self.app.get(self.url, user=user)
         self.assertIn("Personal information", page)
         self.assertNotIn("Delegates", page)
         self.assertIn(user.email, page)
+
+    def test_edit_display_name(self):
+        page = self.app.get(self.url, user=self.responsible)
+        self.assertNotContains(page, "testdisplayname")
+
+        form = page.forms["profile-form"]
+        form["display_name"] = "testdisplayname"
+        form.submit(name="operation", value="profile")
+        self.assertTrue(UserProfile.objects.filter(display_name="testdisplayname").exists())
```

Comment:
should we make sure it doesn't exist beforehand? the name seems like someone else could come up with it independently